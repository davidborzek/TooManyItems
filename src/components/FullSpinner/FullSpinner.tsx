import { ActivityIndicator, StyleSheet, View } from 'react-native';

import React from 'react';

const styles = StyleSheet.create({
  loading: {
    alignItems: 'center',
    flex: 1,
    justifyContent: 'center',
  },
});

/**
 * Ein wiederverwendbarer Component welcher einen Ladespinner bereitstellt.
 */
export default function FullSpinner() {
  return (
    <View style={styles.loading}>
      <ActivityIndicator size="large" />
    </View>
  );
}
