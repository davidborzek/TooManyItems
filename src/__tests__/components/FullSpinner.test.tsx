import FullSpinner from '../../components/FullSpinner/FullSpinner';
import React from 'react';
import renderer from 'react-test-renderer';

it('FullSpinner renders correctly', () => {
  const tree = renderer.create(<FullSpinner />).toJSON();
  expect(tree).toMatchSnapshot();
});
