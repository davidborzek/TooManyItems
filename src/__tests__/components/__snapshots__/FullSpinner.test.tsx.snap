// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FullSpinner renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <ActivityIndicator
    size="large"
  />
</View>
`;
